packages/react-native-jigsaw/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@draftbit/ui",
+  "version": "41.0.0",
```

Comment:
We keep the major version in sync with Expo, so unless I missed SDK 41, you're probably looking at `40.3.0` here.

Reply:
Oh good call actually this is semantically a minor version change anyway come to think of it. 👍